python/lsst/ts/IntegrationTests/base_script.py

```diff
@@ -210,7 +228,7 @@ async def run(self) -> None:
             # This maintains the integrity of the real script_indexes list.
             self.temp_script_indexes = copy.deepcopy(script_indexes)
             # Resume the ScriptQueue to begin script execution.
-            await remote.cmd_resume.set_start(timeout=10)
+            await self.remote.cmd_resume.set_start(timeout=10)
             # Wait for the scripts to complete.
             while not self.all_scripts_done:
                 await asyncio.sleep(0.1)
```

Comment:
I learned you can use `asyncio.sleep(0)` to achieve the same thing. The call still allows other threads to interrupt to do work.